client/src/components/ShippingTable.jsx

```diff
@@ -0,0 +1,163 @@
+            const nextState = shippingStates[nextStateIdx % numStates];
+            elmt.status = nextState;
+          }
+
+          return isTargetId;
+        });
+        shippingStore.setShippingData(shpmtArr);
+        setShpmtData(shpmtArr);
+        const newShpmtStatus = shippingStore.shpmtStatus(shipId);
+        map.setLayoutProperty(
+          mapLayerID + shipId,
+          'visibility',
+          shippingStatesHide.includes(newShpmtStatus) ? 'none' : 'visible'
+        );
+
+        // TODO: Update new changes to DB
+
+        return shpmtArr;
+      });
+    }
+  };
+
+  const handleHover = (e, eType) => {
+    if (map._fullyLoaded) {
+      shippingStore.getDestIds.map((destId) => {
+        const focusId = destId == e.currentTarget.id;
+        const hideStatus = shippingStatesHide.includes(shippingStore.shpmtStatus(destId));
+
+        if (eType == 'over') {
+          map.setLayoutProperty(
+            mapLayerID + destId,
+            'visibility',
+            !focusId || hideStatus ? 'none' : 'visible'
+          );
+        }
+
+        if (eType == 'out') {
+          map.setLayoutProperty(mapLayerID + destId, 'visibility', hideStatus ? 'none' : 'visible');
+        }
+      });
+    }
+  };
+
+  // TODO: Use when Shipping API is complete
+
+  //   if (isLoading) {
+  //     return <Loader />;
+  //   }
+
+  //   if (isSuccess && data.data.length === 0) {
+  //     return (
+  //       <>
+  //         <Heading size="xl" textAlign="center" mt={5}>
+  //           No Shipping
+  //         </Heading>
+  //         <NoResultImage />
+  //       </>
+  //     );
+  //   }
+
+  return (
+    <>
+      <Heading size="xl" textAlign="center" mt={5}>
+        Shipping
+      </Heading>
+      <Table minWidth="unset" width="100%" variant="striped" colorScheme="light">
+        <Thead>
+          <Tr>
+            <StyledTableHeader>Order ID</StyledTableHeader>
+            <StyledTableHeader>Status</StyledTableHeader>
+            <StyledTableHeader>Company</StyledTableHeader>
+            <StyledTableHeader>Location</StyledTableHeader>
+            <StyledTableHeader>Shipping Date</StyledTableHeader>
+            <StyledTableHeader>Delivery Date</StyledTableHeader>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {isSuccess &&
+            shpmtData
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              .map((shipment) => (
+                <Fragment key={shipment._id}>
+                  <StyledTableRow
+                    id={shipment._id}
+                    onMouseOver={(e) => {
+                      handleHover(e, 'over');
+                    }}
+                    onMouseOut={(e) => {
+                      handleHover(e, 'out');
+                    }}
+                  >
+                    <StyledTableCell>{shipment._id}</StyledTableCell>
+                    <StyledTableCell>
+                      <Button id={shipment._id} onClick={handleShpmtStateChange}>
```

Comment:
Could you add a tooltip or something to show what the next status would be? (Like in ClickUp)

![image](https://user-images.githubusercontent.com/4203037/111179887-dfaf9180-8582-11eb-8872-279f36bea90c.png)

Reply:
I will make the requested change